Chore: Minor polishing (#8)
aragon/protocol: README.md

![Aragon Protocol](./docs/aragon-protocol.png)

<img align="right" src="https://github.com/aragon/protocol/workflows/CI/badge.svg">
  <a href="https://github.com/aragon/protocol/actions"/>
</img>

## Project

> #### 👩‍️ [Become an Aragon Protocol guardian](https://ant.aragon.org)

Reviewer: ```suggestion
#### 👩‍️ [Become an Aragon Protocol guardian](https://aragon.org/token/ant)
```

URL might change in the future, but I'd use that one for now :shrug:

Author: Addressed in 6bbd771b6f5d7c3da49fa891bb119e3c16649a20

> Aragon Protocol is now live on Ethereum mainnet. You can become a guardian by staking ANT.

Reviewer: ```suggestion
Aragon Protocol will soon be live on Ethereum mainnet. You can become a guardian by staking ANT.
```

Author: Addressed in 6bbd771b6f5d7c3da49fa891bb119e3c16649a20

#### ⚖ [Check out the Aragon Protocol Dashboard](https://protocol.aragon.org)
The Aragon Protocol Dashboard is the central app where all dispute-related information is available for guardians.

> #### 📚 [Read the User Guide](https://help.aragon.org/category/47-aragoncourt) 

Reviewer: Can probably leave this out for now and add an issue to re-add it later?

Author: Addressed in 6bbd771b6f5d7c3da49fa891bb119e3c16649a20

Read the user guide if you have any doubts about the Aragon protocol, Protocol Dashboard, or related tools.

## Protocol

#### 📓 [Read the full documentation](/docs)

> Aragon Protocol is a dispute resolution protocol that runs on Ethereum. It's one of the core components of the [Aragon Network](https://aragon.org/network/).

Reviewer: ```suggestion
Aragon Protocol is a dispute resolution protocol running on Ethereum. It is the core component of the [Aragon Network](https://aragon.org/network/).
```

😉

Author: Addressed in 6bbd771b6f5d7c3da49fa891bb119e3c16649a20

#### 🚧 Project stage: v1 implementation

> After a long research and development phase, Aragon Protocol's v1 implementation has been [released](https://www.npmjs.com/package/@aragon/protocol) and [deployed](https://etherscan.io/address/0xee4650cBe7a2B23701D416f58b41D8B76b617797#code).

Reviewer: ```suggestion
After a long research and development phase, Aragon Protocol's v1 implementation will soon be released as an upgraded version of Aragon Court.
```

Author: Addressed in 6bbd771b6f5d7c3da49fa891bb119e3c16649a20

> #### ✅ Security review status: audited

Reviewer: Not sure about this one; maybe just refer to the Court's previous audit for now?

Author: Addressed in 6bbd771b6f5d7c3da49fa891bb119e3c16649a20

Aragon Protocol v1 has already been audited by an independent security professional. You can read the audit report [here](https://github.com/gakonst/publications/blob/master/aragon_protocol_audit.pdf). 

#### 👋 Get started contributing with a [good first issue](https://github.com/aragon/aragon-protocol/issues?q=is%3Aissue+is%3Aopen+label%3A%22good+first+issue%22)
Don't be shy to contribute even the smallest tweak. Everyone will be especially nice and helpful to beginners to help you get started!

## Deployed instances

#### Mainnet

The mainnet instance of Aragon Protocol is deployed at [`0xee4650cBe7a2B23701D416f58b41D8B76b617797`](https://etherscan.io/address/0xee4650cBe7a2B23701D416f58b41D8B76b617797#code)

> 

Reviewer: Should we remove this section for now?

Author: Addressed in 6bbd771b6f5d7c3da49fa891bb119e3c16649a20

#### Testing

There are a few testing instances deployed of Aragon Protocol, please refer to the [testing guide](/docs/8-testing-guide) to have a better understanding on using these.

## Help shape Aragon Protocol
- Discuss in [Aragon Forum](https://forum.aragon.org/tags/dispute-resolution)

> - Join the [Aragon Protocol channel](https://discord.gg/nxMejdG) on Discord.

Reviewer: ```suggestion
- Join the [Aragon Discord server](https://discord.com/invite/aragon).
```
